### ISSUE TITLE
Feature Request: applications section: give users the ability to apply for groups within seat

### ISSUE THREAD

**Issue by cryocaustik** [eveseat/seat#301]
group list could be made available to any registered user and the applications section could be used to gather information for various reason (corp recruitment, group qualification, etc. )

**Comment by leonjza** [comment]
What is a group list?

**Comment by cryocaustik** [comment]
group can be anything users define it; just using it as a general term for categorization.

**Comment by leonjza** [comment]
I am sorry, I still don't think I understand what you are saying. Could you rephrase?

**Comment by herpaderpaldent** [comment]
This Feature will be covered with my seat-groups package.
see Wishlistitem and requirements to what i build here: https://github.com/eveseat/seat/issues/234

**Comment by cryocaustik** [comment]
closing this issue as duplicate of #234
